commands.go

```diff
@@ -568,6 +570,32 @@ func (c cmdable) CommandList(ctx context.Context, filter *FilterBy) *StringSlice
 	return cmd
 }
 
+func (c cmdable) CommandGetKeys(ctx context.Context, command string, cmdArgs ...string) *StringSliceCmd {
+	args := make([]interface{}, 3+len(cmdArgs))
+	args[0] = "command"
+	args[1] = "getkeys"
+	args[2] = command
+    for i, cmdArg := range cmdArgs {
```

Comment:
Didn't your editor(ide) help you correct the code formatting? Perhaps you should consider using the gofmt tool.

Reply:
Perhaps you should pay attention to the difference between TAB and SPACE, as this is a common mistake.

You can search on google for replace TAB with SPACE (you ide).